metashare/api/sf_run_flow.py

```diff
@@ -64,15 +64,29 @@ def refresh_access_token(*, config, org_name, scratch_org):
         org_config.config["access_token"] = info["access_token"]
         return org_config
     except HTTPError as err:
-        error_msg = "Are you certain that the org still exists?"
+        error_msg = _("Are you certain that the org still exists?")
 
         if get_current_job():
             job_id = get_current_job().id
-            error_msg += f" If you need support, your job ID is {job_id}."
+            error_msg = _(
+                # This error is user-facing, and so for makemessages to
+                # pick it up correctly, we need it to be a single,
+                # unbroken, string literal (even though adjacent string
+                # literals should be parsed by the AST into a single
+                # string literal and picked up by makemessages, but
+                # that's a gripe for another day). We have relatively
+                # few errors that propagate directly from the backend
+                # like this, but when we do, this is the pattern we
+                # should use.
+                #
+                # This is also why we repeat the first sentence in three
+                # places.
+                f"Are you certain that the org still exists? If you need support, your job ID is {job_id}."  # noqa: E501
```

Comment:
I assume these count as "string literals" even with the `{job_id}` dynamic aspect?

Reply:
Yep. That leaves placeholders for variables for the translator to include.